integration/test_auth.py

```diff
@@ -372,6 +373,37 @@ def test_import_users_with_password(api_key):
     finally:
         auth.delete_user(uid)
 
+@pytest.fixture
+def action_code_settings():
+    return auth.ActionCodeSettings('http://localhost')
+
+def _validate_link_url(link):
```

Comment:
We need to do a stronger check here. See how it's done in Java for example.

https://github.com/firebase/firebase-admin-java/blob/master/src/test/java/com/google/firebase/auth/FirebaseAuthIT.java#L535